boranga/frontend/boranga/src/components/common/conservation_plans_community_dashboard.vue

```diff
@@ -0,0 +1,1203 @@
+        },
+        filterCSCommunityDistrict: function(){
+            let vm = this;
+            vm.$refs.cs_communities_datatable.vmDataTable.ajax.reload(); // This calls ajax() backend call.
+            sessionStorage.setItem(vm.filterCSCommunityDistrict_cache, vm.filterCSCommunityDistrict);
+        },
+        filterCSCommunityEffectiveFromDate: function(){
+            let vm = this;
+            vm.$refs.cs_communities_datatable.vmDataTable.ajax.reload(); // This calls ajax() backend call.
+            sessionStorage.setItem(vm.filterCSCommunityEffectiveFromDate_cache, vm.filterCSCommunityEffectiveFromDate);
+        },
+        filterCSCommunityEffectiveToDate: function(){
+            let vm = this;
+            vm.$refs.cs_communities_datatable.vmDataTable.ajax.reload(); // This calls ajax() backend call.
+            sessionStorage.setItem(vm.filterCSCommunityEffectiveToDate_cache, vm.filterCSCommunityEffectiveToDate);
+        },
+        filterCSCommunityApplicationStatus: function() {
+            let vm = this;
+            vm.$refs.cs_communities_datatable.vmDataTable.ajax.reload(); // This calls ajax() backend call.  
+            sessionStorage.setItem(vm.filterCSCommunityApplicationStatus_cache, vm.filterCSCommunityApplicationStatus);
+        },
+        filterApplied: function(){
+            if (this.$refs.collapsible_filters){
+                // Collapsible component exists
+                this.$refs.collapsible_filters.show_warning_icon(this.filterApplied)
+            }
+        },
+    },
+    computed: {
+        filterApplied: function(){
+            if(this.filterCSCommunityMigratedId === 'all' && 
+                this.filterCSCommunityName === 'all' && 
+                this.filterCSCommunityStatus === 'all' && 
+                this.filterCSCommunityConservationList === 'all' && 
+                this.filterCSCommunityConservationCategory === 'all' && 
+                this.filterCSCommunityRegion === 'all' && 
+                this.filterCSCommunityDistrict === 'all' && 
+                this.filterCSCommunityApplicationStatus === 'all' &&
+                this.filterCSCommunityEffectiveFromDate === '' &&
+                this.filterCSCommunityEffectiveToDate === ''){
+                return false
+            } else {
+                return true
+            }
+        },
+        is_external: function(){
+            return this.level == 'external';
+        },
+        is_internal: function() {
+            return this.level == 'internal'
+        },
+        is_referral: function(){
+            return this.level == 'referral';
+        },
+        addCommunityCSVisibility: function() {
+            let visibility = false;
+            if (this.is_internal) {
+                visibility = true;
+            }
+            return visibility;
+        },
+        datatable_headers: function(){
+            if (this.is_external){
```

Comment:
update community dashboard for conservation plan , the columns looks from conservation status dash